src/components/NotionIntegration.tsx

```diff
@@ -320,15 +354,31 @@ export default function NotionIntegration({
     }
   };
 
+  // Function to show toast for nested pages error
+  const showNestedPagesToast = () => {
+    setToastTitle("Page selection");
+    setToastMessage('Sub-pages are not supported. Please select a different page.');
```

Comment:
the older error message code where the error was shown inside the page itself has not been removed in this PR?